network-api/networkapi/buyersguide/utils.py

```diff
@@ -11,24 +11,9 @@ def get_category_og_image_upload_path(instance, filename):
     )
 
 
-def tri_to_quad(input):
-    if input is True:
-        return 'Yes'
-    if input is False:
-        return 'No'
-    return 'U'
-
-
-def quad_to_tri(input):
-    if input == 'Yes':
-        return True
-    if input == 'No':
-        return False
-    return None
-
-
 class AlterModelBases(ModelOperation):
     """
+    # TODO: This isn't being used anywhere else. Determine if we still need this class.
```

Comment:
at the very least, we want to 100% preserve the knowledge that the fought to gain that necessitated this class, related to renaming models that have subclasses defined (e.g. renaming `Product` when `General(Product)` and `Software(Product)` already exist)